specification/search/data-plane/Azure.Search/preview/2025-05-01-preview/searchservice.json

```diff
@@ -0,0 +1,14715 @@
+          },
+          "description": "Defines combinations of configurations to use with vector search."
+        },
+        "algorithms": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VectorSearchAlgorithmConfiguration"
+          },
+          "description": "Contains configuration options specific to the algorithm used during indexing or querying."
+        },
+        "vectorizers": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VectorSearchVectorizer"
+          },
+          "description": "Contains configuration options on how to vectorize text vector queries."
+        },
+        "compressions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VectorSearchCompressionConfiguration"
+          },
+          "description": "Contains configuration options specific to the compression method used during indexing or querying."
+        }
+      },
+      "description": "Contains configuration options related to vector search."
+    },
+    "AssetProfile": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "externalDocs": {
+            "url": "https://learn.microsoft.com/rest/api/searchservice/Naming-rules"
+          },
+          "type": "string",
+          "description": "The name to associate with this particular asset profile.",
+          "x-nullable": false
+        },
+        "assetType": {
+          "type": "string",
+          "enum": [
+            "path"
+          ],
+          "x-ms-enum": {
+            "name": "AssetType",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "path",
+                "name": "Path",
+                "description": "The part of the Url path after the first compoenet, which is determined by the container name configured in asset store."
+              }
+            ]
+          },
+          "description": "The asset type determines how to locate an asset in the asset store."
+        },
+        "servingType": {
+          "type": "string",
+          "enum": [
+            "sasUrl"
+          ],
+          "x-ms-enum": {
+            "name": "ServingType",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "sasUrl",
+                "name": "SasUrl",
+                "description": "A secure URL that grants restricted access to a resource, allowing users to perform specific actions for a limited period without exposing the account key."
+              }
+            ]
+          },
+          "description": "The serving type determines how to serve assets to API callers."
+        },
+        "servingOptions": {
+          "type": "object",
+          "properties": {
+            "sasExpiry": {
+              "type": "string",
+              "pattern": "^P(?=\\d|T\\d)(\\d+Y)?(\\d+M)?(\\d+W)?(\\d+D)?(T(\\d+H)?(\\d+M)?(\\d+S)?)?$",
```

Comment:
Might be misremembering, but wasn't there some concerns around using ISO8601 duration here? What was the conclusion to that?

Reply:
Asset store was dropped